sharedbuffers/mapped_struct.py

```diff
@@ -344,69 +344,73 @@ def unpack_from(cls, buf, offs, idmap = None):
 
 class proxied_buffer(object):
 
+    HEADER_PACKER = struct.Struct('Q')
```

Comment:
`=` is missing, specifying it explicitly is necessary to avoid unwanted platform variability